bootstrap/backends/ldap/ldap.go

```diff
@@ -18,10 +18,14 @@
 package bsldap
 
 import (
+	"crypto/x509"
 	"fmt"
+	"io/ioutil"
 	"os"
 	"strings"
 
+	"github.com/pkg/errors"
```

Comment:
Assuming we really want this package, an update to go.mod is missing. Please run `go mod tidy`.